src/components/login/login.js

```diff
@@ -0,0 +1,92 @@
+(function () {
+    'use strict';
+
+    var groupSelectionTemplate;
+
+    var showError = function (msg, status) {
+        $('#login-message').text(msg).show();
+    };
+
+    var selectGroup = function (userName, password, account, project, groups) {
+        var dlg = (dlg = $('.group-selection-dialog')).length ? dlg : $(groupSelectionTemplate).appendTo($('body'));
+        var select = $('#log-to-group', dlg);
+        select.find('[value]').remove();
+        $.each(groups, function () {
+            $('<option>')
+                .attr('value', this.groupId)
+                .text(this.name)
+                .appendTo(select);
+        });
+
+        select.on('change', function (e) {
+            $('#forio-login-groupId').val($(e.target).val());
+        });
+
+        var closeDlg = function () {
+            $('.group-selection-dialog').hide();
+        };
+
+        $('.close', dlg).off('click', null).on('click', closeDlg);
+
+        $('.group-selection-dialog').show();
+    };
+
+    $('#login, form:has(#btn-login)').on('submit', function (e) {
+        e.preventDefault();
+        var form = $(e.target);
+        var action = form.attr('action') || 'index.html';
+        var userName = $('#forio-login-username').val();
```

Comment:
Let's just call it username/password etc, "forio-login-username" makes it sound like it needs to be their login for epicenter which is not necessarily true